packages/office-addin-dev-settings/src/cli.ts

```diff
@@ -84,7 +84,8 @@ commander
     .action(commands.webView)
     .on("--help", () => {
         console.log("\nFor [web-view-type], choose one of the following values:\n");
-        console.log("\t'edge' for Microsoft Edge");
+        console.log("\t'edgechromium' for the new Chromium-based Microsoft Edge");
+        console.log("\t'edge' for the older Microsoft Edge");
```

Comment:
leave as is - remove "the older"

Reply:
This should match the other documentation in office-addin-dev-settings once that is decided.